README.md

```diff
@@ -34,6 +34,8 @@ The CI for this repository is written using GitHub Actions. Here is a list of cu
 
 All scripts used in the CI checks are located in the `scripts` folder.
 
+To avoid accidently pushing information about your private gem, configure the `pre-push` git hook by running `git config core.hooksPath .githooks` in your terminal. This git hook will run most of the CI checks and block the push if you don't pass all the tests.
```

Comment:
it's not just about private things. It's also all kind of errors.

Let's also make the command more visible so we found it easily when scorlling:

~~~md
```shell
$ git config core.hooksPath .githooks
```
~~~